README.md

```diff
@@ -116,6 +117,7 @@ Kor provides various subcommands to identify and list unused resources. The avai
 - `replicasets` - Gets unused replicaSets for the specified namespace or all namespaces.
 - `daemonsets`- Gets unused DaemonSets for the specified namespace or all namespaces.
 - `finalizers` - Gets unused pending deletion resources for the specified namespace or all namespaces.
+- `networkpolicies` - Gets unused NetworkPolicies for the specified namespace or all namespaces.
```

Comment:
```suggestion
- `networkpolicy` - Gets unused NetworkPolicies for the specified namespace or all namespaces.
```
Please use singular form for the subcommand, see https://github.com/yonahd/kor/pull/300/commits/1348c487afa59a0f702a52f3da42062f6649ab23.

Reply:
Make sense! Updated :D